### PR TITLE
Add calicoctl/etcdctl/crictl tools for master

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/5216

I don't want to use containers for those debug tools as that adds an additional point of failure in the debug process. But that's just me. Do you think we need containers for those utils?

### CODE REVIEW

File: templates/master.yaml.tmpl [hunk 1]
```diff
@@ -2086,6 +2086,58 @@ storage:
...
+            set -eu
+
+            # download etcdctl
+            ETCD_VERSION=v0.4.5
+            wget https://github.com/etcd-io/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz
+            tar xvf etcd-${ETCD_VERSION}-linux-amd64.tar.gz
+            mv etcd-${ETCD_VERSION}-linux-amd64/etcdctl /opt/bin/etcdctl
+            chmod +x /opt/bin/etcdctl
+            rm etcd-${ETCD_VERSION}-linux-amd64.tar.gz
+            rm etcd-${ETCD_VERSION}-linux-amd64 -Rf
```
Comment (calvix) [inline templates/master.yaml.tmpl]: etcdctl is part of CoreOS distribution so I am not so sure we really need to download yet another etcdctl

File: templates/master.yaml.tmpl [hunk 2]
```diff
@@ -3228,6 +3291,17 @@ systemd:
...
+    enabled: true
+    contents: |
+      [Unit]
+      Description=Calicoctl, etcdctl and crictl tools
+      After=network.target
+      [Service]
+      Type=oneshot
+      ExecStart=/opt/install-debug-tools
+      [Install]
+      WantedBy=multi-user.target
```
Comment (calvix) [inline templates/master.yaml.tmpl]: do we need to install it automatically? from a security perspective, it  would be best to just have an install file that we can run on demand, but that just opinion, not a blocker

File: templates/master.yaml.tmpl (hunk 1 above)
Comment (corest) [reply templates/master.yaml.tmpl]: true

File: templates/master.yaml.tmpl (hunk 2 above)
Comment (corest) [reply templates/master.yaml.tmpl]: That's an additional context you should know. But also no strong opinion on this

**Pull request merged**